src/downloadManager/__init__.py

```
@@ -478,18 +481,25 @@ class CachedImageDownloader(ImageDownloader):
-        return self.images.get(self.image_name(name_or_url))
+        name = self.image_name(name_or_url)
+        try:
+            return self.images[name]
+        except KeyError:
+            return self._try_load_from_disk(name)
+
+    def _try_load_from_disk(self, name: str) -> QPixmap | None:
+        try:
+            pix = QPixmap(self._images_index[name])
+            self.images[name] = pix if self._size is None else pix.scaled(self._size)
+            return pix
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)